src/components/NoteApp/NoteApp.js

```diff
@@ -51,13 +76,44 @@ class NoteApp extends Component {
           alignItems: 'center',
         }}
       >
-        <div className="notes-app">
-          <h2 className="app-header">Notes App</h2>
-          <NoteEditor onNoteAdd={this.handleNoteAdd} />
-          <NoteGrid
-            notes={this.state.notes}
-            onNoteDelete={this.handleDeleteNote}
-          />
+        <div className='notes-app'>
+          <nav className='nav-bar'>
```

Comment:
This can be a standalone component.
you can find a nice example for login component that is nice and clear in this template: [todo-with-identity](https://reshuffle.com/template/todo-with-identity)

Reply:
Will get right to this!